src/std/misc/ref.ss

```diff
@@ -0,0 +1,43 @@
+;;; -*- Gerbil -*-
+;;; (C) vyzo at hackzen.org, proksi at disroot.org
+;;; generic ref
+
+(import :gerbil/gambit
+        :std/generic)
+
+(export ref ~)
```

Comment:
export :ref as well, so that users can customize for their own types.

Reply:
Added